cpp/src/io/parquet/experimental/hybrid_scan_helpers.cpp

```diff
@@ -35,11 +35,27 @@ namespace cudf::io::parquet::experimental::detail {
 using aggregate_reader_metadata_base = parquet::detail::aggregate_reader_metadata;
 using metadata_base                  = parquet::detail::metadata;
 
+using io::detail::inline_column_buffer;
 using parquet::detail::CompactProtocolReader;
 using parquet::detail::equality_literals_collector;
 using parquet::detail::input_column_info;
 using parquet::detail::row_group_info;
 
+namespace {
+
+[[nodiscard]] auto all_row_group_indices(
+  host_span<std::vector<cudf::size_type> const> row_group_indices)
+{
+  std::vector<std::vector<cudf::size_type>> all_row_group_indices;
+  std::transform(row_group_indices.begin(),
+                 row_group_indices.end(),
+                 std::back_inserter(all_row_group_indices),
+                 [](auto rg_indices) { return rg_indices; });
```

Comment:
Is this just a copy?
```suggestion
  std::copy(row_group_indices.begin(),
                 row_group_indices.end(),
                 std::back_inserter(all_row_group_indices));
```

Reply:
could be even less, AFAICT
```
return std::vector<std::vector<cudf::size_type>>(row_group_indices.begin(), row_group_indices.end());
```

Reply:
Yep, updated!